### PR TITLE
DEVEX-75 - Create change log

### PR DESCRIPTION
**Pull request by geetduggal**
@nicolasbockg Please review at your convenience.   @psung @billagee Kurt J.  interest.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Changelog for dx-toolkit releases
+
+This change log spiritually adheres to [these guidelines](http://keepachangelog.com/en/0.3.0/).
+
+Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Security
+
+
+## [Unreleased]
```
Comment (nicolasbockg) [inline CHANGELOG.md]: Is this line placeholder for the actual version number once we do the deployment?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Changelog for dx-toolkit releases
+
+This change log spiritually adheres to [these guidelines](http://keepachangelog.com/en/0.3.0/).
```
Comment (nicolasbockg) [inline CHANGELOG.md]: It looks like a good guideline.

File: CHANGELOG.md (hunk 1 above)
Comment (geetduggal) [inline CHANGELOG.md]: Yes.  This is one of the suggestions to "minimize the effort required" to maintain the Change log buried somewhere in the user guide above.  The idea is when the release is cut, the "Unreleased" is given a version number.

**Comment by nicolasbockg** [comment]
LGTM

**Pull request merged**